main.py

```diff
@@ -109,10 +109,14 @@ async def ranking(msg, *args):
     elif(len(args) == 1 and args[0] == "perfect"):
         await showPerfectRankings(msg.channel)
 
+    elif(len(args) == 3 and args[0] == "swap"):
```

Comment:
Should validate the args to be nicks. Refer to (or abstract out to a function) the functionality of `isMemberShift()`